bundles/core/src/main/java/com/adobe/cq/commerce/core/components/services/urls/UrlProvider.java

```diff
@@ -106,13 +114,21 @@
     String toProductUrl(@Nullable SlingHttpServletRequest request, @Nullable Page page, ProductUrlFormat.Params params);
 
     /**
-     * Returns the product page URL. Only the product identifier must be provided, the implementation will query the needed URL
+     * Returns the product page URL. Only the product identifier must be provided,
+     * the implementation will query the needed URL
      * attributes to generate a complete URL based on the configuration.
      * <p>
-     * This method should be used if the component only can provide the product identifier.
+     * This method should be used if the component only can provide the product
+     * identifier.
+     * 
+     * Either <code>request</code> or <code>request</code> parameter can be
```

Comment:
I guess this means `{@code request} or {@code page}`? Also for all new paragraphs please add a `<p>` tag (no closing one)